src/components/Home.js

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getData } from '../redux/home/home';
+import Card from './Card';
+
+export default function Home() {
```

Comment:
Test cases are missing! Kindly ensure that:

- [ ] The App has unit tests for several components of the app and some pure functions in Redux code.
- [ ]  App has integration tests using React Testing Library
   * You can perform a mocking data fetching and Redux Store connection.

Reply:
Issue well Addressed :+1: